convert_to_hdf5/test_depth.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -o errexit
+set -o pipefail
+
+
+
+data="/media/gsuveer/391cd01c-d5a2-4313-947a-da8978447a80/gsuveer/Desktop/Flo_data"
```

Comment:
hard coded data.... look at the other bash files to see how to take in args. If this file is just for testing and no longer needed, then remove it.

Reply:
Removed